frontend/eigen_generic/include/algebra/eigen_generic.hpp

```diff
@@ -1,127 +0,0 @@
-/** Algebra plugins library, part of the ACTS project
- *
- * (c) 2020-2024 CERN for the benefit of the ACTS project
- *
- * Mozilla Public License Version 2.0
- */
-
-#pragma once
-
-// Project include(s).
-#include "algebra/math/eigen.hpp"
```

Comment:
You were a bit over-eager with this one as well I fear. 🤔

Let's discuss which frontends we want to get rid of. I don't want to throw out the baby as well in all of this...